src/pages/Swap.tsx

```diff
@@ -163,21 +162,21 @@ function Swap(): ReactElement {
         ? (
             formState.currentSwapPairs
               .map(({ to, type: swapType }) => {
-                if (!TOKENS_MAP?.[to.symbol]) {
+                if (!tokensMap?.[to.symbol]) {
                   console.log("unknown symbol", { to, swapType })
                   return null
                 }
-                const { symbol, name, decimals } = TOKENS_MAP[to.symbol]
-                const amount = tokenBalances?.[symbol] || Zero
+                const token = tokensMap?.[to.symbol]
```

Comment:
We know that `token` is defined based on the check at 165, thus this should be `const token = tokensMap[to.symbol] as BasicToken` and remove all of the `?? ""` below